app/views/messages/new.html.erb

```diff
@@ -4,13 +4,18 @@
 
 <%= content_for :messages do %>
   <div class="absolute flex flex-col items-center justify-between w-full h-[96%] lg:max-w-[700px] xl:max-w-[810px]">
-    <div class="flex flex-row items-center h-full">
-      <span class="text-2xl font-semibold tracking-tighter">
-        How can I help you today?
-      </span>
+    <div class="z-0 flex flex-row items-center h-full overflow-y-auto scrollbar-hide">
+      <div class="flex flex-col items-center gap-3">
+        <div class="p-3 bg-white border border-gray-100 rounded-full">
+          <%= inline_svg("logo_iso", width: "24", height: "24", class: "text-gray-950") %>
+        </div>
+        <span class="text-2xl font-semibold tracking-tighter">
+          How can I help you today?
+        </span>
+      </div>
     </div>
 
-    <div class="grid w-full grid-cols-1 gap-2 px-6 md:grid-cols-2">
+    <div class="z-10 grid w-full grid-cols-1 gap-2 px-6 bg-transparent md:grid-cols-2">
       <%= render partial: "messages/main_column/conversation_starter",
```

Comment:
@florianfelsing I just tried out this PR locally and it looks great! Thanks so much for taking the lead on all this. It's surprising how much a logo & the introduction of a brand color begin to up the professionalism of the app.

My one question: what's the introduction of the `z-10` and `z-0` do? I always get a little wary of introducing explicit z-index, but I know we do end up needing it sometime. I was curious and tried removing these in my dev environment and I didn't see a change in any appearance (i tried light / dark and different viewport sizes). Do you think we need the z's?

Reply:
You're right, we didn't really need them here in this case - removed them.